src/components/SignUpForm/index.js

```diff
@@ -73,15 +113,22 @@ const SignUpForm = ({ onSubmit }) => {
         testID="confirm-password-input"
         {...inputProps(FIELDS.passwordConfirmation)}
       />
+
+      <Picker
+        label={strings.SIGN_UP.gender}
+        items={genderItems}
+        placeholder={PLACEHOLDER_PICKER}
+        touchableWrapperTestId="gender-picker"
+        {...pickerProps(FIELDS.gender)}
+      />
+
       <ErrorView errors={{ error }} />
-      <View style={styles.button}>
-        <Button
-          testID="signup-submit-button"
-          title={status === LOADING ? strings.COMMON.loading : strings.SIGN_UP.button}
-          onPress={handleSubmit}
-          disabled={formHasErrors}
-        />
-      </View>
+      <Button
+        testID="signup-submit-button"
+        title={status === LOADING ? strings.COMMON.loading : strings.SIGN_UP.button}
+        onPress={handleSubmit}
+        disabled={formHasErrors}
```

Comment:
I'll suggest you to 
```suggestion
        disabled={loading || formHasErrors}
```

Reply:
since you have a defintion for `status === LOADING` you can create a `const` and reuse it on both props